src/localplayer.h

```diff
@@ -145,7 +145,7 @@ class LocalPlayer : public Player
 
 private:
 	// clang-format off
-	void accelerateHorizontal(const v3f &target_speed, f32 max_increase, bool slippery);
+	void accelerateHorizontal(const v3f &target_speed, f32 max_increase);
```

Comment:
Oops! I thought I had removed that. Will fix later.

Thanks for the clang hint.